### ISSUE TITLE
Unfortunate license requirement

### ISSUE THREAD

**Issue by eliasnaur** [EsotericSoftware/spine-runtimes#147]
I don't think the benefits of the recent requirement of purchasing a Spine license outweighs the negative.
1. The runtimes are not open source anymore. The license may be "liberal" and similar to NBSD, but is now a completely non standard license. With the previous standard license, the games created with a spine runtime were clear of legal issues. Now there is the additional legal burden of a non standard license.
2. Not being open source is not fair to the community. There are already quite a few third party runtimes (I'm contributing to one of them), and I'm sure there has been outside contributions to the official runtimes. You created the official runtimes to avoid the chicken-and-egg problem of a new tool, but as Spine grows in popularity the runtimes will probably be more and more maintained by the community.
3. You argue that the runtimes could be used with a competing product. That is true, but I think that's a weak argument. I'm willing to bet that the vast majority of your efforts go towards Spine itself, so any competing tool won't gain much from open source runtimes. And even if a competing tool end up using your runtimes, so what? That will only bolster the credibility of Spine and make the file formats standard. It's not like the market is overflowing in quality 2d animation tools.

In short, have faith in your ability to build and maintaining an excellent tool and keep the runtimes clean and simple licensed. If you really want to complicate the licensing for little gain, do it when you're already industry standard not when Spine is still gaining traction.

**Comment by NathanSweet** [comment]
We are happy to answer questions about licensing. It has not yet been an issue.

An engine developer could make the part of their engine that uses Spine a separate module, so users that don't use it are unaffected by the Spine runtime license. EDIT: Spine licensing is now more friendly for OSS developers!
http://esotericsoftware.com/blog/Our-new-licensing-explained
http://esotericsoftware.com/spine-editor-license
http://esotericsoftware.com/spine-runtimes-license

The Spine runtime is very core functionality in many games that use it. Not providing the source is a significant hindrance for integration of any tool, no matter how much value you think the tool provides.

I encourage you to evaluate the official runtimes yourself. They are designed to be low level and flexible enough to handle many use cases.
